src/kernel/capinv.c

```diff
@@ -550,18 +550,16 @@ cap_thd_op(struct cap_thd *thd_cap, struct thread *thd, struct pt_regs *regs,
 }
 
 /**
- * Process the send event, and notify the appropriate end-points.
+ * Notify the appropriate end-points.
  * Return the thread that should be executed next.
  */
 static struct thread *
-asnd_process(struct thread *rcv_thd, struct thread *thd, struct tcap *rcv_tcap,
+notify_process(struct thread *rcv_thd, struct thread *thd, struct tcap *rcv_tcap,
```

Comment:
Okay, will do that. 
I was going for `notif_process` initially because `notification_process` sounded long. I'll change it.